packages/frontend/src/components/campaigns/campaign/rewards/styles.module.css

```diff
@@ -11,7 +11,7 @@
 }
 
 .breakdownContainer {
-    @apply flex flex-col gap-3 p-3;
+    @apply flex flex-col gap-3 p-3 shadow-lg px-4 py-4;
```

Comment:
We should avoid the shadow, since I don't think we have that anywhere else; let's keep the overall padding to `p-3`, it's consistent with the other popovers (or we need to change that in the others).